price/pricing.go

```diff
@@ -0,0 +1,71 @@
+package price
+
+import (
+	"fmt"
+	"github.com/singnet/snet-daemon/blockchain"
+	"github.com/singnet/snet-daemon/handler"
+	log "github.com/sirupsen/logrus"
+	"math/big"
+	"strings"
+)
+
+type Pricing struct {
```

Comment:
What do you think about renaming it to `PricingConfig` or `PricingStrategy` (as component name below is `PricingStrategy`)

Reply:
Done